build/tasks/aria-supported.js

```diff
@@ -0,0 +1,59 @@
+/*eslint-env node */
+'use strict';
+
+const aQ = require('aria-query');
+const { roles, aria: props } = require('aria-query');
+
+module.exports = function (grunt) {
+	grunt.registerMultiTask(
+		'aria-supported',
+		'Task for generating a diff of supported aria-roles.',
+		function () {
+			const entry = this.data.entry;
+			const destFile = this.data.destFile;
+			const listType = this.data.listType.toLowerCase(); // enum = 'supported', 'unsupported', 'all'
+
+			const axe = require('../../axe');
```

Comment:
Can this live at the top level? I'm guessing not, because that file probably doesn't exist yet. Please add a comment to explain why this is here.